README.md

```diff
@@ -1,50 +1,77 @@
-Edit the quota for `In-use IP addresses` in the [quota page](https://console.cloud.google.com/iam-admin/quotas) or with the following `gcloud` command.
+2. Enable Billing for the new Project
 
 ```bash
-gcloud alpha services quota update \
-    --service=compute.googleapis.com --consumer=projects/<your-project-name> \
-    --metric=compute.googleapis.com/global_in_use_addresses \
-    --unit=1/{project} --value=64
+gcloud alpha billing projects link <unique-project-id> --billing-account=$(gcloud alpha billing accounts list --format=json | jq '.[]."name"'  | tr -d '"' | awk -F'/' '{ print $2}')
 ```
-\_Note: If the increase isn't immediate or you receive an error, contact GCP support/account management\_
+
+2. Enable the APIs required by the deployment: [`cloudresourcemanager.googleapis.com`, `compute.googleapis.com`].
+
+```bash
+gcloud services enable cloudresourcemanager.googleapis.com compute.googleapis.com
+```
+
+3. Service Quota increase: `In-use IP addresses (default is 8)` has too low of a default value for deploying a Kubernetes beyond 3 nodes and quota should be increased to 64 to create headroom in available IPs.
+Edit the quota for `In-use IP addresses` in the [quota page](https://console.cloud.google.com/iam-admin/quotas).  Be sure to increase the quota for the region relevant to the new project.
 
 ![Screenshot of Quota Increase](https://user-images.githubusercontent.com/6570292/210277244-f3a75d06-763f-4bdc-805e-4f8bd3c77ad5.png)
 
 ### Service Account
 
 1. [Create a service account](https://console.cloud.google.com/iam-admin/serviceaccounts/create) and [download the json key file](https://console.cloud.google.com/iam-admin/serviceaccounts/details/101612329871957262389/keys), save the file as `creds.json`
 
+Create the service account:
+```bash
+gcloud iam service-accounts create <service-account-name> \
+    --description="<service-account-description>" \
+    --display-name="<service-account-display-name>"
+```
+
+Create the json key file:
+```bash
+gcloud iam service-accounts keys create creds.json \
+    --iam-account=<service-account-name>@<unique-project-id>.iam.gserviceaccount.com
+```
+
+Download the key file and place it in the root directoy of the new directory created for this deployment.
+
 2. Set env variable while in the directory with the `creds.json`:
 
     ```bash
 
     export GOOGLE_CREDENTIALS=$(pwd)/creds.json
     ```
 
-3. In the project, [grant the service account the `Owner` role](https://console.cloud.google.com/iam-admin/iam) by selecting `GRANT ACCESS` at the top of the screen and filling out the principal (your service account role) and the `Owner` role.
+3. In the project, [grant the service account the `Owner` role](https://console.cloud.google.com/iam-admin/iam) by selecting `GRANT ACCESS` at the top of the screen and filling out the principal (service account role) and the `Owner` role.
```

Comment:
Since you do this below with the CLI I think we should just remove this?

Reply:
ya, sounds good

Reply:
I'll leave the links in as docs, but remove the instructions